src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/roleassignment/AmRoleAssignmentServiceConsumerTestForCreateRoleAssignment.java

```diff
@@ -0,0 +1,154 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.roleassignment;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.Pact;
+import au.com.dius.pact.consumer.dsl.DslPart;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.model.RequestResponsePact;
+import com.google.common.collect.Maps;
+import io.restassured.http.ContentType;
+import net.serenitybdd.rest.SerenityRest;
+import org.assertj.core.api.Assertions;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.wataskmanagementapi.SpringBootContractBaseTest;
+
+import java.util.Map;
+
+import static io.pactfoundation.consumer.dsl.LambdaDsl.newJsonBody;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+
+public class AmRoleAssignmentServiceConsumerTestForCreateRoleAssignment extends SpringBootContractBaseTest {
+
+    private static final String RAS_CREATE_ROLE_ASSIGNMENT_URL = "/am/role-assignments";
+
+    @Pact(provider = "am_role_assignment_service_create", consumer = "am_org_role_mapping")
```

Comment:
please change the consumer name as it is over writing our pact in the remote broker

Reply:
Hi @ConorH994  sorry about that done now